_posts/12-05-01-Building-your-Application.md

```diff
@@ -4,7 +4,7 @@ title: Izgradnja i isporučivanje vaše aplikacije
 anchor: building_and_deploying_your_application
 ---
 
-## Izgradnja i isporučivanje vaše aplikacije {#build_title}
+## Izgradnja i isporučivanje vaše aplikacije {#building_and_deploying_your_application}
 
 Ako zateknete sebe da ručno radite promene šeme baze podataka ili ručno pokrećete testove pre nego što ažurirate svoje
 fajlove (ručno), razmislite još jednom! Uz svaki dodatni zadatak koji se izvodi ručno potreban da razvijete novu verziju

```

Comment:
Ovde mi se nešto ne slaže sa originalom: "With every additional manual task needed to deploy a new version of your app", a mi smo ga preveli kao: "Uz svaki dodatni zadatak koji se izvodi ručno potreban da razvijete novu verziju aplikacije". Ja bih celu tu rečenicu ovako preveo: "Svaki dodatni ručni zahvat tokom deploy-a nove verzije vaše aplikacije povećava šanse za fatalne greške."